openapi-contracts/src/test/resources/domains/conversation/templates/v1/TemplateTranslationDto.json

```diff
@@ -0,0 +1,13 @@
+{
+  "language_code": "fr-FR",
+  "content": "template translation content value",
```

Comment:
For educational purpose, it would be better to set a valid content for this parameter such as `{\"text_message\":{\"text\":\"Message from a template v1.\"}}`

Reply:
Done